en_US/configuration/dashboard.md

```diff
@@ -0,0 +1,36 @@
+# Dashboard
+
+In EMQX, Dashboard is a web-based graphic interface to manage and monitor EMQX and connected devices in real time. 
+
+For example,  configure a listener for EMQX Dashboard for accepting all incoming connections.
+
+```
+dashboard {
+  listener {
+    bind  =  "0.0.0.0:18083"
+    max_connections  =  512
+    ssl_options {
+      cacertfile = "etc/certs/cacert.pem"
+      certfile = "etc/certs/cert.pem"
+      keyfile = "etc/certs/key.pem"
+    }
+  }
+}
+```
+
+Where,
+
+- `bind  =  "0.0.0.0:18083"`  is to set the network address and port number that the listener will bind to. In this case, the listener will bind to all available network interfaces (`0.0.0.0`) on port `18083`.
+- `max_connections  =  512` is to set the maximum number of concurrent connections that the listener will accept. In this case, the maximum number of connections is set to `512`.
+- `ssl_options` is the SSL/TLS configuration option for the listener, it has three properties:
+  - `cacertfile`: This sets the path to the file containing the trusted CA (certificate authority) certificates that the listener uses to verify the authenticity of the client certificates.
+  - `certfile`: This sets the path to the file containing the SSL/TLS certificate for the listener.
+  - `keyfile`: This sets the path to the file containing the private key corresponding to the SSL/TLS certificate.
+
+:::tip
+
+To add a listener via Dashboard, click **Configuration** -> **Listeners **on the left navigation menu of the Dashboard. Once you configured these items with the Dashboard, your settings will override the same configuration items in `emqx.conf`.
+
```

Comment:
I think this is a bit confusing. The `dashboard listener` is somewhat unique and can not be configured through the dashboard itself (correct me if I'm wrong). Those listeners above are listeners for MQTT client connections.

Reply:
@wivwiv can you confirm, I think we can add more listeners on the dashboard

Reply:
The Dashboard's listener (http 18083) is not an MQTT Listener, it is configured in `dashboard.listeners.{http,https}`.

The PM group will discuss where in the dashboard to provide the configuration page